client-ui/src/app/app.component.html

```diff
@@ -15,6 +15,8 @@
   -->
 
 
-<app-header></app-header>
+<app-header *ngIf="(isEmbeddedMode$ | async) === false"></app-header>
```

Comment:
Why not simply `*ngIf="!(isEmbeddedMode$ | async)"`?

Reply:
Because it is easy to make a mistake and to miss "!".

That is why when evaluating:

true:

`*ngIf="isEmbeddedMode$ | async"`

false:

`*ngIf="(isEmbeddedMode$ | async) === false"`